src/test/java/in/projecteka/devservice/bridge/BridgeTest.java

```diff
@@ -54,4 +64,35 @@ void shouldUpdateBridgeUrl() {
                 bridgeRequest.getUrl(),
                 session.getTokenType() + " " + session.getAccessToken());
     }
+
+    @Test
+    void shouldUpdateBridgeServiceEntry() {
```

Comment:
You can change test name to shouldPopulateBridgeServiceEntry or shouldUpsertBridgeServiceEntry as it is doing both add & update

Reply:
done.changed to shouldPopulateBridgeServiceEntry